### PR TITLE
Net7 sample update with script

### PR DESCRIPTION
**Pull request by jpalac**

### CODE REVIEW

**Comment by danielmarbach** [comment]
FYI I sent in https://github.com/Particular/docs.particular.net/pull/5958

Would it maybe make sense for you to also remove netcoreapp3.1 since it is out of support effectively today and you are already touching all TFMs?

**Comment by jpalac** [comment]
> FYI I sent in #5958
> 
> Would it maybe make sense for you to also remove netcoreapp3.1 since it is out of support effectively today and you are already touching all TFMs?

We will look into doing that - will get back to you tomorrow if it poses unexpected issues.

File: samples/azure-functions/service-bus/ASBFunctions_2/AzureFunctions.Messages/AzureFunctions.Messages.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+	<TargetFrameworks>net7.0;net6.0</TargetFrameworks>
```
Comment (abparticular) [inline samples/azure-functions/service-bus/ASBFunctions_2/AzureFunctions.Messages/AzureFunctions.Messages.csproj:3]: Should we be changing netstandard to net for message definitions?

File: samples/azure-functions/service-bus/ASBFunctions_3/AzureFunctions.Messages/AzureFunctions.Messages.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+	<TargetFrameworks>net7.0;net6.0</TargetFrameworks>
```
Comment (abparticular) [inline samples/azure-functions/service-bus/ASBFunctions_3/AzureFunctions.Messages/AzureFunctions.Messages.csproj:3]: Should we be changing netstandard to net for message definitions?

**Comment by danielmarbach** [comment]
I'm done making your life harder ;) No more groundbreaking changes

**Review by tmasternak — APPROVED** [review]

**Pull request merged**